tests/casp/stack_controller.pm

```diff
@@ -70,15 +70,17 @@ sub velum_bootstrap {
     assert_screen 'velum-bootstrap-page';
     barrier_wait "WORKERS_INSTALLED";
 
+    # Calculate position of master node radio button
+    my $needle = assert_screen('master-checkbox-xy')->{area};
+    my $row    = $needle->[0];                                  # get y-position of master node
+    my $col    = $needle->[1];                                  # get x-position of checkbox
+    my $x      = $col->{x} + int($col->{w} / 2);
+    my $y      = $row->{y} + int($row->{h} / 2);
+
     # Select master and bootstrap
-    send_key "ctrl-f";
-    type_string "master.op\n";
-    send_key "esc";
-    send_key "tab";
-    send_key "spc";
-    assert_screen "master-selected";
+    mouse_set $x, $y;
+    mouse_click;
```

Comment:
assert_and_click is actually supposed to return the mouse to the last mouse_set - which would be the checkbox